devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy

```diff
@@ -247,15 +249,30 @@ def zwaveEvent(switchcolorv3.SwitchColorReport cmd) {
 		result << createEvent(name: "color", value: hexColor)
 		// Send the color as hue and saturation
 		def hsv = rgbToHSV(*colors)
-		result << createEvent(name: "hue", value: hsv.hue)
-		result << createEvent(name: "saturation", value: hsv.saturation)
-		// Reset the values
-		RGB_NAMES.collect { state.colorReceived[it] = null}
+		if (state.setColor.red == state.colorReceived.red && state.setColor.green == state.colorReceived.green && state.setColor.blue == state.colorReceived.blue) {
```

Comment:
Now I'm confused. Wasn't the original issue that you couldn't count on `state.colorReceived` to be correct?

Reply:
I think the key part of it, was setting it back to _null_ here https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/34180/files#diff-923fb700963e1771dfa88fd274738f37L253 
Now, even if for a first report there is an invalid value in `state` map, querying it once again, generally solves problem

Reply:
so do we still want to restrict this to only the values we're setting? If we can avoid it I'd like to in order to preserve inter-operation with other z-wave controllers